class/main.rb

```diff
@@ -0,0 +1,6 @@
+require './app'
```

Comment:
- [ ] As pointed out by the previous reviewer, I still can't run your project and it throws me the same error as he mentioned before

Reply:
## Hello @SirriRyisa 
- [ ] I am still getting the same error, please when importing locally try using `require_relative` instead of `require`